### PR TITLE
PDP: Data Proxy usage notifications

### LINKED ISSUE #3851: PDP: Notifications for Data Proxy usage

### PR DESCRIPTION
**Pull request by zachtil**
## Describe this PR

PDP docs update for automated Data Proxy usage notifications

## Changes

* New page Billing > Data Proxy usage notifications
* Updates to Billing > Invoices
* Updates to Billing > Plans and quotas

## What issue does this fix?

Fixes #3851

### CODE REVIEW

**Review by jasonkuhrt — APPROVED** [review]

**Review by marcoieni — APPROVED** [review]
looks great!

File: content/800-data-platform/850-billing/06-data-proxy-usage.mdx [hunk 1]
```diff
@@ -0,0 +1,35 @@
...
+
+<TopBlock>
+
+Prisma Data Platform continuously monitors the Data Proxy usage of your projects and sends automated notifications to the email address of your GitHub account when your project reaches usage milestones.
+
+</TopBlock>
+
+## Milestones that trigger notifications
+
+You receive automated email notifications based on your project plan and on the amount of Data Proxy usage you reach during any given month.
```
Comment (peterskim12) [inline content/800-data-platform/850-billing/06-data-proxy-usage.mdx]: Update beginning of sentence to "You will receive...". Thanks!

Comment (zachtil) [reply content/800-data-platform/850-billing/06-data-proxy-usage.mdx]: Thank you for the review, Peter.

We have a style guide rule to always use present simple tense. 

https://www.prisma.io/docs/about/prisma-docs/style-guide/writing-style#use-the-present-simple-tense

I've reworded the sentence to indicate that PDP sends the automated email notifications. Let me know if you have feedback for the change.

Comment (peterskim12) [reply content/800-data-platform/850-billing/06-data-proxy-usage.mdx]: Thanks for the rewording. That sounds better. I'm not an English language grammar expert -- but I think changing the subject from "You" to "Prisma Data Platform" makes it sound more correct when using present simple tense. I appreciate that we have a style guide and we adhere to it! Thanks!

**Pull request merged**